src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,136 @@
+package com.wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity(name = "book")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(name = "genre")
```

Comment:
There is no need to specify the name if it is the same than the property name. You would only need it with `sub_title`, and even that field could be renamed `subtitle` as a single word to avoid the mapping.

Reply:
Solved, i removed property "name" from @column annotation properties